### PR TITLE
Minor changes to bikes edit and feedbacks show for deleted bikes

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
Image 8-7-19 at 3 45 PM
Image 8-7-19 at 3 44 PM

File: app/views/admin/bikes/_bike.html.haml [hunk 1]
```diff
@@ -111,6 +111,15 @@
                   - if bike.creation_state.bulk_import.ascend?
                     %small
                       Ascend
+          %tr
```
Comment (sethherr) [inline app/views/admin/bikes/_bike.html.haml]: The display should be much more prominent when the bike is deleted. And it doesn’t need to show up at all when it is t deleted.

File: app/views/admin/feedbacks/show.html.haml [hunk 2]
```diff
@@ -44,7 +44,12 @@
               %pre
                 %code
                   = @feedback.feedback_hash.to_yaml.gsub("---\n", "")
-
+        - if @feedback.feedback_type == "bike_delete_request"
+          %tr
+            %td
+              View Bike
+            %td
+              = link_to "Deleted Bike", edit_admin_bike_path(id: @feedback.feedback_hash["bike_id"])
```
Comment (sethherr) [inline app/views/admin/feedbacks/show.html.haml]: can you show if the bike is actually deleted before you click this link?

File: app/controllers/admin/feedbacks_controller.rb [hunk 3]
```diff
@@ -11,6 +11,9 @@ def index
 
   def show
     @feedback = Feedback.find(params[:id])
+    if @feedback.feedback_type == "bike_delete_request"
+      @bike = Bike.unscoped.find(@feedback.feedback_hash["bike_id"])
```
Comment (sethherr) [inline app/controllers/admin/feedbacks_controller.rb]: `find` will throw an error if the bike is truly deleted.

File: app/views/admin/bikes/edit.html.haml [hunk 4]
```diff
@@ -16,6 +16,11 @@
   .col-12
     %h1
       Edit Bike
+    - if @bike.deleted?.present?
+      %h2.text-danger
+        Deleted at:
+        %span.text-danger.convertTime
```
Comment (sethherr) [inline app/views/admin/bikes/edit.html.haml]: No need for text danger on this too.

File: app/controllers/bikes_controller.rb [hunk 5]
```diff
@@ -28,6 +28,9 @@ def index
 
   def show
     @components = @bike.components
+    if @bike.deleted?
```
Comment (sethherr) [inline app/controllers/bikes_controller.rb]: This should actually go in the `find_bike` method which already has something similar

**Pull request merged**